specs/backlog.md

```diff
@@ -0,0 +1,92 @@
+# Creation of Backlog as a Ranked List
+
+We want to ensure that decisions about what to work on incorporate prioritization's agreed by the the team, product owners, our partners and customers. Specifically, we want to reach the point that:
+
+- Prioritization is clear so that decisions and trade-offs may take an issues priority into account.
+	- this should be visible by a Rank Order List and, where appropriate, an explanation of the prioritization (example below).
+- The teams priorities are visible to interested parties. 
+- Items "ready to be worked on" state is easily identifiable.
+- The tool/process we use has a connection to GitHub.
```

Comment:
Do you want to elaborate here on what is implied by "a connection to GitHub"?  Do you mean "the tool/process we use should react automatically to changes in Github issues and pull requests"?

Reply:
Resolved in doc.